scripts/e2etests/VSUtils.ps1

```diff
@@ -55,11 +55,20 @@ function GetVSIDEFolderPath
 
 function KillRunningInstancesOfVS
 {
-    $processName = 'devenv'
-    Write-Host "Kill any running instances of $processName..."
-    (Get-Process "$processName" -ErrorAction SilentlyContinue) | Kill -ErrorAction SilentlyContinue
-
-    WaitForProcessExit -ProcessName "$processName" -TimeoutInSeconds 30
+    Get-Process | ForEach-Object {
+        if(-not [string]::IsNullOrEmpty($_.Path)) {
+            $processPath = $_.Path | Out-String
+            if($processPath.StartsWith("C:\Program Files (x86)\Microsoft Visual Studio", [System.StringComparison]::OrdinalIgnoreCase))
+            {
+                Write-Host $processPath
+                Stop-Process $_ -ErrorAction SilentlyContinue -Force
+                if($_.HasExited) {
+                Write-Host "Killed process:" $_.Name
```

Comment:
> $_.Name [](start = 44, length = 8)

Use $processPath if you are already assigning it.

Reply:
i actually did it deliberately so i can see both values

Reply:
Why would they be different?